### PR TITLE
Add PostgreSQL support for performance tests

### PR DESCRIPTION
**Pull request by hwupathum**
### Summary 

This pull request introduces support for PostgreSQL databases in the deployment setup. The changes span multiple files and include creating PostgreSQL databases, adding PostgreSQL connectors, and updating configuration scripts to handle PostgreSQL. Below are the most important changes:

### Related issue

https://github.com/wso2/product-is/issues/21509

### CODE REVIEW

File: common/is/clean-database-postgres.sql [hunk 1]
```diff
@@ -0,0 +1,11 @@
+-- add tables that need to be truncated
+TRUNCATE TABLE IDN_AUTH_SESSION_STORE CASCADE;
+TRUNCATE TABLE IDN_AUTH_TEMP_SESSION_STORE CASCADE;
```
Comment (Sachin-Mamoru) [inline common/is/clean-database-postgres.sql:3]: Can you change the existing mysql script also to the same naming convention?

Comment (Sachin-Mamoru) [reply common/is/clean-database-postgres.sql:3]: Additionally let's add this inside cleanup script directory inside this is directory

**Comment by Sachin-Mamoru** [comment]
And did you discuss whether to support the performance tests with JDK 17 and 21 as well?

**Comment by hwupathum** [comment]
> And did you discuss whether to support the performance tests with JDK 17 and 21 as well?

This is handled in a separate effort

**Review by Sachin-Mamoru — APPROVED** [review]

**Review by asha15 — APPROVED** [review]

**Pull request merged**